views/homepage.handlebars

```diff
@@ -10,13 +10,12 @@
     <section class="description">
         <div class="large-6 columns">
             <h2>Where Great Teams Are Made</h2>
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet quisquam inventore quod velit eos consequuntur
-            neque tempore blanditiis atque officia! Reiciendis eum aliquid tempora fugiat et quam, asperiores sit
-            voluptatem?
+            Are you tired of not being able to build an all-star Pokemon team? Well, we have the solution right here! Introducing
+            Team PokeWise! Here you can create any team you'd like from any version of the most popular game in the world.
         </div>
         <div class="large-6 columns">
-            <h1>Ready to ?</h1>
-            <a href="#" class="button">Let's Get Started</a>
+            <h1>Ready to become a Pokemon legend?</h1>
+            <a href="controllers/generator-page-routes.js" class="button">Let's Get Started</a>
```

Comment:
The href here should be "/generator-page" because it goes based off of the base url, which in this case is just "http://localhost:3001".